rnd/autogpt_server/autogpt_server/server/integrations.py

```diff
@@ -80,6 +84,85 @@ async def callback(
     )
 
 
+@integrations_api_router.get(
+    "/{provider}/credentials", response_model=list[CredentialsMetaResponse]
+)
+async def list_credentials(
+    provider: Annotated[str, Path(title="The provider to list credentials for")],
+    user_id: Annotated[str, Depends(get_user_id)],
+    store: Annotated[SupabaseIntegrationCredentialsStore, Depends(get_store)],
+) -> list[CredentialsMetaResponse]:
+    credentials = store.get_creds_by_provider(user_id, provider)
+    return [
+        CredentialsMetaResponse(credentials_id=cred.id, credentials_type=cred.type)
```

Comment:
Do we need to return anything else here?

Reply:
- Authorized scopes
- Some kind of human-readable identifier, e.g. a description or the login/e-mail of the linked account

Reply:
Done - scopes and email